portfolio/src/main/webapp/script.js

```diff
@@ -49,6 +54,11 @@ function getServerName() {
     });
 }
 
+function getUserInput() {
+    var input = document.getElementById("text-area").value;
```

Comment:
No need to set this to a variable since it's only being used once. You can just directly return this line.

Note: If you do need to set a variable. Using `const` or `let` is preferred in es6.

Reply:
In this case just use const because the value does not change